### ISSUE TITLE
How to train and deploy YOLO Segmentation with cityscapes?

### ISSUE THREAD

**Issue by NJUSTghw** [eric612/MobileNet-YOLO#97]
demo_cityscapes.sh: 1: demo_cityscapes.sh: ./examples/yolo/yolo_detect: not found

**Comment by eric612** [comment]
1. [Make lmdb](https://github.com/eric612/MobileNet-YOLO/tree/master/data/cityscapes)
2. Use these [models](https://github.com/eric612/MobileNet-YOLO/tree/master/models/cityscapes) to train

However , it is not complete tutorial , you may meet some problems and feel free to ask question here.

**Comment by kkjong** [comment]
Hi, @eric612 
I tried to train and deploy YOLO segmentation with cityscapes.
I created lmdb with leftImg8bit images, xml annotations and binary label images.
When I run  mobilenet_yolov3_train.prototxt with pre-trained model you uploaded, I found mismatch num_output in conv26. In test.prototxt, conv26 has **num_out : 19**, but, in train.prototxt, conv26 has **num_out : 3**.
So, I chagned train.prototxt num_out : 19 to num_out : 3 in conv26, I succeded to train .

But, demo was something wrong.
I upload my result:
![Selection_001](https://user-images.githubusercontent.com/52026501/63836131-f21fa580-c9b3-11e9-9db1-cb58084f2ade.png)
![Selection_002](https://user-images.githubusercontent.com/52026501/63836215-1a0f0900-c9b4-11e9-9d7e-a5111f4c5167.png)

I think object detection is working well, But segmentation has some problem.

**Comment by eric612** [comment]
@kkjong 
If you want to train models , recommend to use imagenet pre-trained. And the channel numbers was based on your [lablemap setting](https://github.com/eric612/MobileNet-YOLO/blob/master/data/cityscapes/labelmap_seg.prototxt)  

But here has a problem , the deploy model which I gave was 3 class segmentation , however , the default label map setting and trainning prototxt was 19 class segmentation , you need modify this problems , for example :
* Change label map setting and modify trainning prototxt 'conv26' channel 19 to 3 
* Or change deploy prototxt  'conv26'  channel to 19

**Comment by BaofengZan** [comment]
@kkjong Hi,  Have you successfully trained the model? When I was running, the loss was always high.
Have you encountered this problem?
